--- a/src/host/layer23/src/common/subscriber.c
+++ b/src/host/layer23/src/common/subscriber.c
@@ -930,7 +930,7 @@
 		plmn = talloc_zero(ms, struct gsm_sub_plmn_list);
 		if (!plmn)
 			return -ENOMEM;
-		osmo_plmn_to_bcd(&data[0], &plmn->plmn);
+		osmo_plmn_from_bcd(&data[0], &plmn->plmn);
 		llist_add_tail(&plmn->entry, &subscr->plmn_list);
 
 		LOGP(DMM, LOGL_INFO, "received PLMN selector (mcc-mnc=%s) from SIM\n",
@@ -1027,7 +1027,7 @@
 		na = talloc_zero(ms, struct gsm_sub_plmn_na);
 		if (!na)
 			return -ENOMEM;
-		osmo_plmn_to_bcd(&data[0], &na->plmn);
+		osmo_plmn_from_bcd(&data[0], &na->plmn);
 		LOGP(DMM, LOGL_INFO, "received Forbidden PLMN %s from SIM\n",
 			osmo_plmn_name(&na->plmn));
 		na->cause = -1; /* must have a value, but SIM stores no cause */
